rustio/src/client.rs

```diff
@@ -93,10 +97,16 @@ impl Client {
         }
     }
 
-    pub fn get_playable_station_url(&self, station: &Station) -> String{
+    pub fn play_station(&self, station: &Station,mut atomic_playbin: Mutex<gstreamer::Element>) {
```

Comment:
Rustio and Gradio are two crates, which are completely independently to each other. The rustio crate can be used by other applications too.

So it doesn't make sense here to return a gstreamer element. It forces the other application to use gstreamer too.

Rustio is just a API abstraction to radio-browser.info.